### ISSUE TITLE
508-defect-2 [KEYBOARD]: Loading indicator should not have a tabindex="0"

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#32577]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah

## Details

Generally speaking, non-interactive elements should not be placed in the natural tab order of a page. Doing so may confuse keyboard users as:
- It does not have a halo indicating its received focus, making it seem like an invisible tab stop
- It may give them the false impression they can take action on it

There may be times where sending focus to the loading indicator will be necessary, so using a `tabindex="-1"` would be preferred as it allows the element to receive focus without being added into the natural tab order of the page.

## Acceptance Criteria

- [ ] Tabindex is equal to "-1" not "0"

**Comment by bkjohnson** [comment]
Solved in the linked PR
